samples/nrf9160/nrf_cloud_mqtt_multi_service/child_image/mcuboot/boards/nrf9160dk_nrf9160.conf

```diff
@@ -12,3 +12,7 @@ CONFIG_SPI_NRFX_RAM_BUFFER_SIZE=32
 CONFIG_PM_OVERRIDE_EXTERNAL_DRIVER_CHECK=y
 
 CONFIG_FPROTECT=y
+
+# Enabling SPI increases the MCUBoot image size such that it won't fit in the default secondary
+# partition size (0xC000). The minimum required size is 0xD000
+CONFIG_PM_PARTITION_SIZE_MCUBOOT=0xD000
```

Comment:
That's interesting -- I'm surprised you don't also have to make the same change for the main image as well.

Reply:
Maybe I do and I've missed something important; What do you mean by the main image?

Reply:
the child_image/ folder applies to, well, the child image, surprisingly.  In this case, the child image is mcuboot.  That is built and linked separately from the main image, which in this case is mmss.  The partition manager combines the output of the linked images to a single .bin, .hex, etc.

Reply:
It looks like both the main and secondary partitions share this size. I will verify by doing an A-B FOTA test (I think I already did, but I cannot remember, so better to do it again and be certain)

I will also update the changelog and commit message to accurately reflect this.

Reply:
Oh, you meant "main image" as in "not the child image". I thought you mean "main image" as in "the non-secondary mcuboot partition", my bad.

I guess the worry, here, then, might be that we're overflowing the total available flash on 9160 if we don't use external flash for the secondary partition? I'll check for that as well

Reply:
Both APP and bootloader FOTA appear to function as expected, and the partition regions are not overflowing the available flash, since for each MCUBoot partition there is an empty partition that shrinks by the same amount that the MCUboot partition grew, so I think we're good